eoscostarica/eosio-dashboard
# fix(webapp): remove duplicated nodes text

### remove duplicated nodes text

### What does this PR do?

- Resolve #N/A

### Steps to test

1. make run lacchain
2. navigate to http://localhost:3000/
3. check card titles

#### CheckList
